x/btcstaking/keeper/msg_server.go

```diff
@@ -315,7 +314,6 @@ func (ms msgServer) AddBTCDelegationInclusionProof(
 		btcutil.NewTx(stakingTx),
 		btccParams.BtcConfirmationDepth,
 		btcDel.StakingTime,
-		minUnbondingTime,
```

Comment:
I think @RafilxTenfen comment is valid here - https://github.com/babylonlabs-io/babylon/pull/289#issuecomment-2491815295, ie we should pass here the `params.MinUnbondingTimeBlocks`  from the params used when creating the delegation, not retrieve latest one from the store inside the function